components/infobox/commons/infobox_league.lua

```diff
@@ -41,6 +41,7 @@ function League.run(frame)
 end
 
 function League:createInfobox()
+	self.args.abbreviation = self:_fetchAbbreviation()
```

Comment:
Move this below frame imo

Reply:
done, even though the frame gets kicked with the other PR ;)